team-validator.js

```diff
@@ -62,17 +62,11 @@ class Validator {
 			}
 		}
 
-		for (let i = 0; i < format.teamBanTable.length; i++) {
-			let bannedCombo = true;
-			for (let j = 0; j < format.teamBanTable[i].length; j++) {
-				if (!teamHas[format.teamBanTable[i][j]]) {
-					bannedCombo = false;
-					break;
-				}
-			}
```

Comment:
The latest commit doesn't remove the tables, it just moves them into the banlist table.